### PR TITLE
Main README cleanup

### PR DESCRIPTION
**Pull request by yangyansong-adbe**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by cacheung — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -66,15 +66,29 @@ Initializing the SDK should be done in native code inside your `AppDelegate` (iO
 @import AEPServices;
 @import AEPSignal;
 @import AEPLifecycle;
-@import AEPIdentity;
-@import AEPUserProfile;
+@import AEPEdge;
```
Comment (cacheung) [inline README.md:69]: How about add // AppDelegate.h before the import statement, so it is clearer  for use  to know where to add  it.

File: README.md [hunk 2]
```diff
@@ -66,15 +66,29 @@ Initializing the SDK should be done in native code inside your `AppDelegate` (iO
...
+@import AEPEdgeIdentity;
+@import AEPEdgeConsent;
+//@import AEPMessaging;
+//@import AEPUserProfile;
+//@import AEPAssurance;
+//@import AEPIdentity;
 ...
 @implementation AppDelegate
 -(BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
```
Comment (cacheung) [inline README.md:78]: And add // AppDelegate.m here

File: README.md [hunk 3]
```diff
@@ -94,13 +108,16 @@ Initializing the SDK should be done in native code inside your `AppDelegate` (iO
 ###### **Android:**
 ```java
 import com.adobe.marketing.mobile.AdobeCallback;
-import com.adobe.marketing.mobile.Identity;
 import com.adobe.marketing.mobile.InvalidInitException;
-import com.adobe.marketing.mobile.Lifecycle;
 import com.adobe.marketing.mobile.LoggingMode;
-import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Lifecycle;
```
Comment (cacheung) [inline README.md:113]: Add //MainApplication.java at the top of the import, so user know it is for MainApplication file?

**Review by cacheung — COMMENTED** [review]
looks good! few comments

**Comment by yangyansong-adbe** [comment]
@cacheung  I think we don't need to provide the file names for the sample code as SDK users' mobile projects may have different setup with different file/class names.

**Pull request merged**